tests/test_tokenization_common.py

```diff
@@ -3325,6 +3326,11 @@ def test_save_pretrained(self):
                 tokenizer_r_files = tokenizer_r.save_pretrained(tmpdirname2)
                 tokenizer_p_files = tokenizer_p.save_pretrained(tmpdirname2)
 
+                # make sure that all ".json" files are saved in the correct format
+                for file_path in tokenizer_r_files + tokenizer_p_files:
```

Comment:
pretty aggressive addition to test that should make sure all tokenizer json files will from now on have the correct format

Reply:
Smart!